guidance/src/guidance/guidance_state_machine.cpp

```diff
@@ -108,7 +108,7 @@ namespace guidance
 
     void GuidanceStateMachine::onRouteEvent(const cav_msgs::RouteEventConstPtr& msg)
     {
-        if(msg->event == cav_msgs::RouteEvent::LEFT_ROUTE ||
+        if(msg->event == cav_msgs::RouteEvent::ROUTE_DEPARTURE ||
```

Comment:
since all the route events are past tense, suggesting to make this ROUTE_DEPARTED instead of route departure.